certs/certs.c

```diff
@@ -28,7 +28,10 @@ const char certificates[] =
 "ksLi4xaNmjICq44Y3ekQEe5+NauQrz4wlHrQMz2nZQ/1/I6eYs9HRCwBXbsdtTLS\r\n"
 "R9I4LtD+gdwyah617jzV/OeBHRnDJELqYzmp\r\n"
 "-----END CERTIFICATE-----\r\n"
-/*DigiCert Global Root CA*/
+#ifndef LEAN_CERTS
+//Note: for devices with limited resources, other certs are defined out to save space. 
```

Comment:
>other certs are defined out to save space [](start = 44, length = 41)

this can be "developers can choose to remove certs not relevant to their region. Note that removing these might cause issues if the device were to be moved into/from Germany or China". Since they aren't automatically defined out.